### PR TITLE
fix: fixes ImportConflict NPE [DHIS2-12162]

### PR DESCRIPTION
**Pull request by gnespolino**

### CODE REVIEW

File: dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/importsummary/ImportConflictTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ */
+package org.hisp.dhis.dxf2.importsummary;
+
+import org.junit.Test;
+
+public class ImportConflictTest
+{
+
+    @Test
+    public void testImportConflictObjectCanBeNull()
```
Comment (teleivo) [inline dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/importsummary/ImportConflictTest.java:36]: Can you please add a test for the other case when message is null, now that you added the test class 😋

Comment (gnespolino) [reply dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/importsummary/ImportConflictTest.java:36]: the message should never be null... I can add the test case but I don't see if it makes much sense. However it wont harm so...

Comment (teleivo) [reply dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/importsummary/ImportConflictTest.java:36]: true, so the test should test `requireNonNull( message );`

**Review by teleivo — APPROVED** [review]

**Review by jbee — APPROVED** [review]
